src/SamplesApp/UITests.Shared/Windows_UI_Xaml_Controls/TimePicker/Sample1.xaml

```diff
@@ -15,10 +15,20 @@
         <controls:SampleControl.SampleContent>
             <DataTemplate>
                 <Grid Background="White">
-                    <TimePicker ios:FlyoutPlacement="Full"
+                    <Grid.RowDefinitions>
+                        <RowDefinition Height="Auto" />
+                        <RowDefinition Height="*" />
+                    </Grid.RowDefinitions>
+                    	<Button Content="Set To Current Time"
+							Command="{Binding SetToCurrentTime}" Margin="10" HorizontalAlignment="Center" Visibility="Collapsed"/>
+                    <TimePicker Grid.Row="1"
+                                Margin="15,10"
+                                ios:FlyoutPlacement="Full"
                                 HorizontalAlignment="Center"
-                                MinuteIncrement="5"
-                                ClockIdentifier="12HourClock">
+                                VerticalAlignment="Top"
+                                DataContext="{Binding Time, Mode=TwoWay}"
```

Comment:
Why do you need this binding?

Reply:
this and the settocurrent time curently collapsed button was to test the two way binding that actually doesnt refresh from viewmodel to the control for some unknown reason..

Reply:
I think we should have another sample that binds Time, as binding Time instead of DataContext is the typical use-case.

Reply:
yes it should bound to Time that's for sure i'll fix it =)